### ISSUE TITLE
PowerSlit does not have members for width or center.

### ISSUE THREAD

**Issue by silkenelson** [pcdshub/pcdsdevices#470]
The class for the power slit does not have members/functions for the widths and centers.
## Expected Behavior
I would expect for a slit to have gaps/widths and offsets/centers. Ideally, <slit>.() would return the opening size and <slit>(#) would make a square opening and <slit>(#1,#2) would set the horizontal. vertical opening to these numbers. I would like any slit to behave like the old JJ slits in HXR do.

## Current Behavior
It only seems to have the motor as member.

## Possible Solution
Implement missing functions?

## Steps to Reproduce (for bugs)
Look at the code.

## Context
I'm trying to prepare for the HXR FEE commissioning and would like to have a beam line python that behaves as the scientists might reasonably expect.

## Your Environment
xpp3 as xppopr on xpp-daq.

**Comment by ZLLentz** [comment]
This shouldn't be too hard to fix, the PVs do exist

**Comment by ZLLentz** [comment]
I took a stab at this today. cc @ghalym 

Originally I thought we could sensibly merge/subclass the new slits with the old code that behaves as desired and everything would work out, but the PVs/behavior don't exactly match 100% 1-to-1.

We will need to do essentially the same thing as the old class, except:
- All PVs have a `_RBV` version for reading and a non-`_RBV` version for writing, so they'll need to be set up as such so we see the readbacks.
- The `:DMOV` for checking when the combined motion is done is missing. We'll have to use a combination of the individual axes `.DMOV` instead, or the state move `:DMOV_RBV` if that works better. It would need some testing.
- The old slits moved immediately on getting a value, the new ones want you to put to a start-moving PV. We will need to identify and include the start-moving PV and wrap it into the move method in a sensible way.
- The `BLOCKED` PV is missing altogether, but I don't think I miss it much.

**Comment by ghalym** [comment]
Thank you @ZLLentz for looking at this.
I was planning to make a change to eliminate the start-moving PV, and make them move on a value change. Is that also what you meant here by wrapping it into the move method?

**Comment by ZLLentz** [comment]
@ghalym Yes, that's exactly what I mean. I was going to do it on the Python side, but of course doing it on the PLC is better so it happens in your PyDM screen as well.
